pkg/ebpf/events_pipeline.go

```diff
@@ -380,15 +380,19 @@ func (t *Tracee) matchPolicies(event *trace.Event) uint64 {
 	return bitmap
 }
 
-func parseContextFlags(flags uint32) trace.ContextFlags {
+func parseContextFlags(containerId string, flags uint32) trace.ContextFlags {
 	const (
 		contStartFlag = 1 << iota
 		IsCompatFlag
 	)
-	return trace.ContextFlags{
-		ContainerStarted: (flags & contStartFlag) != 0,
```

Comment:
Why not just do
`ContainerStarted: (flags & contStartFlag) != 0 && containerId != ""` here instead? Simple one line change.

Reply:
Firstly I did as you suggested but I thought that any change in any field related to that return would change the other lines in the future. So I extracted their assignment to single lines (which won't be automatically realigned in a single change).

![image](https://github.com/aquasecurity/tracee/assets/3372117/5be78fc5-90fa-426b-a579-837bca7a392d)